import org.apache.hadoop.io.retry.RetryPolicies.FailoverOnNetworkExceptionRetry;
public class FunctionUse1897 {
public void funcUse() {
FailoverOnNetworkExceptionRetry failoveronnetworkexceptionretry = new FailoverOnNetworkExceptionRetry();
failoveronnetworkexceptionretry.shouldRetry(null,null,null,null);
}
}